screens/media/CreatePost.tsx

```diff
@@ -39,6 +49,9 @@ const CreatePost = () => {
   const [isPickingVideo, setIsPickingVideo] = useState<boolean>(false);
   const [imageContent, setImageContent] = useState<LazyStaticImage[]>([]);
   const [isPickingImages, setIsPickingImages] = useState<boolean>(false);
+  const [selectedRoute, setSelectedRoute] = useState<Route | undefined>(
```

Comment:
TIL: 
`const [a, setA] = useState<T | undefined>(undefined)`
is equivalent to 
`const [a, setA] = useState<T>()`
Obviously not breaking, but it's way cleaner :)